### PR TITLE
Update schema

### LINKED ISSUE #200: Change input field from samplesheet from "sequence" to "id"
### Description of feature

In most of the pipelines is used as "id" and is also stored in meta as "id". I find it a bit confusing - maybe it is better to use id from the beginning.

### PR DESCRIPTION
**Pull request by luisas**
Fixes #200 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If necessary, also make a PR on the nf-core/proteinfold _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.

### CODE REVIEW

**Comment by luisas** [comment]
this won't run until this is merged: https://github.com/nf-core/test-datasets/pull/1362

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
LGTM!
Please udpate the changelog and once tests pass we are ready to merge :rocket

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
LGTM!
Please udpate the changelog and once tests pass we will be ready to merge 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Thank you to everyone else that has contributed by reporting bugs, enhancements
 - [[PR ##163](https://github.com/nf-core/proteinfold/pull/163)] - Fix full test CI.
 - [[#150]](https://github.com/nf-core/proteinfold/issues/150)] - Add thanks to the AWS Open Data Sponsorship program in `README.md`.
 - [[PR ##166](https://github.com/nf-core/proteinfold/pull/166)] - Create 2 different parameters for Colabfold and ESMfold number of recycles.
+- [[PR ##205](https://github.com/nf-core/proteinfold/pull/205)] - Change input schema from sequence,fasta to id,fasta.
```
Comment (JoseEspinosa) [inline CHANGELOG.md:75]:
```suggestion
- [[PR ##205](https://github.com/nf-core/proteinfold/pull/205)] - Change input schema from `sequence,fasta` to `id,fasta`.
```

**Review by JoseEspinosa — APPROVED** [review]
Awesome!

**Pull request merged**